Mappers/DbMapper.php

```diff
@@ -199,7 +202,13 @@ public function get(array $identifiers) : ArraySerializable
             $conditions[] = new EqualCriteria($indName, $indValue);
         }
 
-        return $this->load(new SearchCriteriaConstructor($conditions, 1))->getFirst();
+        $entity = $this->load(new SearchCriteriaConstructor($conditions, 1))->getFirst();
+
+        if (is_null($entity)) {
+            throw new EntityNotFoundException();
```

Comment:
This exception will end up with 500 status code

Reply:
This exception will end up with 404 status code, pls see on Application class

Reply:
ok